src/App.jsx

```diff
@@ -22,10 +24,28 @@ function App() {
 
 	useEscKey(() => setShowEventDetails(false));
 
+	const [popupPosition, setPopupPosition] = useState({})
+
+	const createPopupPosition = (event)=>{
```

Comment:
This should be wrapped in a `useMemo`

https://react.dev/reference/react/useMemo#usememo

Reply:
Is that possible since the function takes `event` as argument?

Reply:
is it possible since the function takes `event` as argument?

Reply:
Yes, sorry, you're right. A refactor would only catch the use case of clicking the same event twice, which is unnecessary.

Reply:
That's totally cool. I tried to modify the function to fit useMemo in, but it complicated the things even more. Let's keep it for now and refactor it in future maybe.